### ISSUE TITLE
Is this snap stale?

### ISSUE THREAD

**Issue by chrfle** [davel/jira-cli-snap#1]
At the time of writing:

This snap was last updated to 1.4.0 in September 2023.

The jira-cli project released 1.5.1  January 2024.

**Comment by davel** [comment]
> At the time of writing:
> 
> This snap was last updated to 1.4.0 in September 2023.
> 
> The jira-cli project released 1.5.1 January 2024.

Hello,

I believe it's alive still! I have built 1.5.1. Could you install it and confirm that it works for you? Something like `sudo snap install jira-cli --channel edge`

All the best,
Dave

**Comment by chrfle** [comment]
> > At the time of writing:
> > This snap was last updated to 1.4.0 in September 2023.
> > The jira-cli project released 1.5.1 January 2024.
> 
> Hello,
> 
> I believe it's alive still! I have built 1.5.1. Could you install it and confirm that it works for you? Something like `sudo snap install jira-cli --channel edge`
> 
> All the best, Dave

It works, however, why is the binary named "jira-cli" and not "jira"? The tool itself thinks it should be called jira:
```
❯ jira-cli
Interactive Jira command line.

USAGE
  jira <command> <subcommand> [flags]
```

**Comment by davel** [comment]
> It works, however, why is the binary named "jira-cli" and not "jira"? The tool itself thinks it should be called jira:

https://snapcraft.io/docs/snapcraft-app-and-service-metadata#appsapp-name

Snap requires that the name of binary begins with the name of the Snap, `jira-cli`. I don't think we could call it `jira`, because it isn't.
